app/controllers/sessions_controller.rb

```diff
@@ -114,4 +114,15 @@ def login(options = {})
 
     redirect_to_origin_or_default
   end
+
+  def find_or_create_user_from_hash(hash)
+    # first user will be promoted to super admin
+    hash[:role_id] = Role::SUPER_ADMIN.id unless User.where.not(email: 'seed@example.com').exists?
```

Comment:
Can you use User.count??.. for any reason if I delete the 'seed@example.com', then all the user's created after that will be Super admin.

Reply:
This just checks if there are any users _not_ named `seed@example.com`. Deleting a user with that name wouldn't result in all users being given super admin privileges.

Reply:
oh!! yes. Missed the `not`.